### DISCUSSION TITLE
Seeking help: Issues with Kohya SS GUI on Google Colab for LoRA creation

### DISCUSSION THREAD

**Discussion by Atsushi888** [bmaltais/kohya_ss#3025] [Q&A]
Hello everyone,

I’m trying to use Kohya SS GUI on Google Colab to create LoRA models. While I managed to get the GUI to launch, I’m encountering multiple errors when attempting to proceed with the actual LoRA creation process.

I have tried addressing these errors through various means (details can be provided later), but so far, I haven’t been able to resolve all the issues.

If anyone has experience running Kohya SS GUI in a similar setup or has insights into potential fixes, I would greatly appreciate your input. I’m committed to solving this issue and am open to collaborating on troubleshooting.

Thank you in advance for your time and assistance!

**Comment by Sylsatra** [comment]
Please provide your config and errors. It is impossible to help without knowing the exact details of what happened.

**Comment by thuansd** [comment]
I am trying to do the same, and I appear to be stuck at load_tokenizer (train_util.py).  The last message I see on the console is "prepare tokenizer" and it is just stuck there briefly, then the training button switched from "Stop Training" back to "Start Training" at which point the training is over.

**Reply by thuansd** [reply]
I am going to investigate whether the "prepare tokenizer" is just taking a long time, and gradio just losses contact, as the cell did not terminate, and was still running.

**Reply by thuansd** [reply]
Nope, the Colab GPU runtime became idle shortly after the console stopped outputting so no dice.
